app/folder/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { Footer } from "@/src/components/Footer";
+import { Header } from "@/src/components/Header";
+import LinkManager from "@/src/components/LinkManager/LinkManager";
+import { END_POINT } from "@/src/constants";
+import getFormattedLinks from "@/src/utils/getFormattedLinks";
+import { useRef } from "react";
+// import { useFetchData } from "@/src/hooks/useFetchData";
+// page 컴포넌트 안에서 useFetchData 사용시 eslint - hooks rule에 저촉
+
+async function getFetchData(url: string) {
+  try {
+    const res = await fetch(url);
+    const result = await res.json();
+    return result;
+  } catch (error) {
+    if (error instanceof Error) {
+      // alert(error.message);
+      console.error(error.message);
+    }
+  }
+}
+
+// use Hooks 사용 불가
+export default async function page() {
+  // links data fetch
+  const linksData = await getFetchData(END_POINT.LINKS);
```

Comment:
이렇게 표현되는게 저는 더 좋을 것 같습니다~
```suggestion
  const linksData = await getLinks();
```